config/redis_store.yml

```diff
@@ -6,9 +6,10 @@ development:
   <<: *default
   namespace: <%= ENV.fetch('REDIS_NAMESPACE') { 'scalelite_development' } %>
 
+
 test:
   <<: *default
-  driver: null # Reset to allow fakeredis to load
```

Comment:
Need to discuss if we should be using fakeredis or not.

Reply:
After discussion with @jfederico , we decided it would be better not to change any testing configs to ensure the functionality of Minitest for the near future.  

Allowing the RSpec test data to be established on Redis default `db:0` requires no changes in the configs.